jemma.osgi.javagal/src/main/java/org/energy_home/jemma/javagal/layers/PropertiesManager.java

```diff
@@ -148,7 +148,14 @@ public int getKeepAliveThread() {
 	}
 	
 	
-	
+	public Boolean getzgdDump() {
+        	String value = props.getProperty("dump");
+        	return (!value.equalsIgnoreCase("0"));
```

Comment:
why using a numeric value for the "dump" variable instead of "true/false" ?

Also: it is also possible the value is not specified - in that case it is implicitly off - the code should account for that.